components/badges.css

```diff
@@ -0,0 +1,151 @@
+@import url(/main.css);
+
+.main-container {
+  display: flex;
+  gap: auto;
+}
+
+.sidebar {
+  height: 900px;
+  background-color: black;
+  padding: 10px;
+  width: 25%;
+  display: fixed;
+}
+
+.sidebar h2 {
+  margin-left: 25px;
+  display: flex;
+  text-align: left;
+  color: #ebe8e8;
+}
+
+li a {
+  font-size: 1.4rem;
+  color: white;
+  height: 3rem;
+  text-align: center;
+  margin-top: 18px;
+  display: flex;
+  margin-left: 30px;
```

Comment:
Better to use rems instead of px, and shorthand property for margin

Reply:
Noted.